GameServerLib/Logic/GameObjects/AttackableUnits/AI/Minion.cs

```diff
@@ -97,52 +95,36 @@ public override void Update(float diff)
 
                 if (ScanForTargets()) // returns true if we have a target
                 {
-                    KeepFocussingTarget(); // fight target
+                    //Here we need to add another check: Positioning.
+                    if (RecalculateAttackPosition())
+                    {
+
```

Comment:
Is this functionality for a later pull request or unfinished functionality for this pull request?